lib/rules/no-duplicate-attr-inheritance.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @fileoverview Disable inheritAttrs when using v-bind="$attrs"
+ * @author Hiroki Osame
+ */
+'use strict'
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'enforce inheritAttrs: false when using v-bind="$attrs"',
+      category: 'recommended',
```

Comment:
Please change category to undefined
```suggestion
      category: undefined,
```